pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -498,6 +517,36 @@ private BrokerResponseNative handleSQLRequest(long requestId, String query, Json
     return brokerResponse;
   }
 
+  private BrokerResponseNative processExplainPlanQuery
+      (BrokerRequest brokerRequest, BrokerRequest offlineBrokerRequest, BrokerRequest realtimeBrokerRequest,
+          String offlineTableName, String realtimeTableName) {
+      QueryContext queryContext = BrokerRequestToQueryContextConverter.convert(brokerRequest);
```

Comment:
I found out that `queryContext` is also converted in `reduceOnDataTable` method of `BrokerReduceService` class on broker. Maybe we should move the conversion upfront so that the same object can be reused?